From: Frederic Bonnard <frediz@linux.vnet.ibm.com>
Date: Wed, 21 Feb 2018 15:36:39 +0100
Subject: direct-systemd-to-always-restart-opal-prd

---
 external/opal-prd/opal-prd.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/external/opal-prd/opal-prd.service b/external/opal-prd/opal-prd.service
index 9956912..a53d8d9 100644
--- a/external/opal-prd/opal-prd.service
+++ b/external/opal-prd/opal-prd.service
@@ -5,6 +5,7 @@ ConditionPathExists=/sys/firmware/devicetree/base/ibm,opal/diagnostics
 
 [Service]
 ExecStart=/usr/sbin/opal-prd --pnor /dev/mtd0
+Restart=always
 
 [Install]
 WantedBy=multi-user.target
